blocks/pocketguide/pocketguide.css

```diff
@@ -0,0 +1,44 @@
+.pocketguide {
+    text-align:center;
+    position:relative;
+    padding-top:136px;
+    border-radius:10px;
+    border:1px solid #7f5732;
+    margin:100px 15px;
+    min-width: 200px;
+}
+
+.pocketguide>*>*>h2 {
+    text-transform: none;
+}
+
+.pocketguide>div>div>p:nth-child(4)>picture{
```

Comment:
Are we expecting more than one picture? If not, I think it makes more sense to simplify this selector to just target the picture.